notebooks/examples/1 _INTRO_MNIST.ipynb

```diff
@@ -0,0 +1,871 @@
+{
```

Comment:
- This notebook is modified: `notebooks/examples/2_Sentiment140.ipynb` => could you please remove the save method
- 2 csv are added: `notebooks/examples/testdata.manual.2009.06.14.csv` & `notebooks/examples/training.1600000.processed.noemoticon.csv` => could you please update the `.gitignore` file with `*.csv`
- And the `data_distribution.png` is created here: `distributed-learning-contributivity/notebooks/examples/C:\GitHub\distributed-learning-contributivity\experiments\nlp/scenario_1_repeat_1_2020-08-24_16h01_c37/graphs` => Could you please remove this *black magic* which is related to your local directories and the `experiments` path!

Reply:
1. Dropped
2. Done
3. Done

Reply:
Thanks ! The default `experiment_path` will be `./experiment` for now